data_demos/NF_hackathon_exomeseq_datademo.Rmd

```diff
@@ -0,0 +1,216 @@
+
+```{r load data, eval=TRUE, results='show', message=FALSE, warning=FALSE}
+
+#here() # check working directory
+entity <- synGet("syn19955828")  # download file 
+exome_data <- read_csv(entity$path) # read csv file 
+
+```
+# Exploring the data files
+
+## Structure of the file
+
+```{r explore data, eval=TRUE, results='show', message=FALSE, warning=FALSE, include=TRUE}
+
+##Inspect the file
+print("Dimensions of the file:")
+dim(exome_data)
+
+print("Contents of the file:")
+head(exome_data)
+
+```
+
+The following table contains descriptions of some of the metadata variables included in the data file:
+
+Var | Description
+--- | --- 
+ _Hugo_Symbol_  |  names of genes according to HUGO database
+ _Entrez_gene_Id_  |  Gene ID according to Entrez Database
+ _NCBI_Build_ | Reference genome that was used to align the exomeSeq data
+ _Chromosome_ | Chromosome number (range 1-22 and X,Y),  Chr M == mitochondrial genome (absent in exomes with NCBI Build == hg19)
+ _species_  |  the source of the specimen
+ _age_  |  the age of the patient 
+ _sex_  |  the sex of the patient
+ _tumorType_  |  the the type of tumor, can be one of 7 different diagnoses
+ _isCellLine_  |  indicates whether the origin tissue was a cell line or a patient
+ _study_  |  the specific initiative/consortia that the study was a part of
+
+For more information regarding the various columns in the dataset please visit here: https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/#vep
+
+## Metadata 
+
+The metadata in a healthcare study includes descriptions of the specimens, and the details of the studies they originated from. We can explore the metadata by visualizing various aspects of it to get a sense of the data contained within our dataset and put it in context of the disease and patients.
+
+For example, we can visualize the data distribution according to sex.
+
+```{r explore Metadata1, eval=TRUE, results='show', message=FALSE, warning=FALSE, include=TRUE}
+
+#Plot Metadata
+exome_data %>% group_by(Tumor_Sample_Barcode) %>%
+ggplot(aes(x=sex))+
+    geom_bar(aes(x=sex,fill=as.factor(tumorType)),position='dodge')+
+    ggtitle('Exome-seq sample distribution')
+```
+
+We can also look at the different types of mutations observed for various genes according to the tumor type
+
+```{r explore Metadata2, eval=TRUE, results='show', message=FALSE, warning=FALSE, include=TRUE}
+#Plot genes with metadata
+analyzeMetdataWithGenes<-function(datafile,prefix, gene_a, ...){
+  #look at some marker genes using normalized zScore
+  ggplot(subset(datafile,Hugo_Symbol%in%c(gene_a, ...)))+
+    geom_jitter(aes(x=Hugo_Symbol, y=Variant_Classification, shape=study,col=tumorType), width =0.1, size=2, height = 0.1)+
+    ggtitle(paste(prefix,'Variant types in selected genes', sep = '::'))
+}
+analyzeMetdataWithGenes(exome_data, "NF_hackathon", "NF1", "NF2", "GFAP", "ASTE1")
```

Comment:
can you add some simple context for why you picked these genes (targeted at a non biological crowd)